apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/store/db/DbScanStoreMetrics.scala

```diff
@@ -36,7 +36,9 @@ class DbScanStoreMetrics(metricsFactory: LabeledMetricsFactory) extends AutoClos
       -1L,
     )(MetricsContext.Empty)
 
-  val cache = new CacheMetrics(prefix :+ "cache", metricsFactory)
```

Comment:
this rename is technically breaking but the metrics didn't even work before so seems fine